doc/source/raysgd/v2/api.rst

```diff
@@ -48,32 +46,39 @@ HorovodConfig
 
 .. autoclass:: ray.sgd.HorovodConfig
 
+
+Callbacks
+---------
+
 .. _sgd-api-callback:
 
 SGDCallback
------------
+~~~~~~~~~~~
 
-.. autoclass:: ray.sgd.SGDCallback
+.. autoclass:: ray.sgd.callbacks.SGDCallback
     :members:
```

Comment:
Can we define this at the top-level?

Not strongly committed to this opinion, I just want to differentiate this as an abstract class as opposed to a directly usable callback.

Reply:
Changed back!